client/src/pages/recording-view/hooks/use-spectrogram.tsx

```diff
@@ -1,9 +1,8 @@
 import { useGetIQData } from '@/api/iqdata/Queries';
-import { useMeta } from '@/api/metadata/queries';
-import { useMemo, useState } from 'react';
+import { useMemo } from 'react';
 import { useSpectrogramContext } from './use-spectrogram-context';
 
-export function useSpectrogram() {
+export function useSpectrogram(currentFFT) {
```

Comment:
where does this get used?

Reply:
line 28:
      requiredBlocks.push(currentFFT + i * (fftStepSize + 1)

Reply:
Was this broken before?

Reply:
No before it would hold this state inside the hook and expose methods to set it but since this state it is used in other components makes sense to move it outside of the hook